src/services/AbstractDFG.jl

```diff
@@ -783,6 +775,10 @@ function getIncidenceMatrixSparse(dfg::G; solvable::Int=0)::Tuple{SparseMatrixCS
     return adjMat, varLabels, factLabels
 end
 
+function getAdjacencyMatrixSparse(dfg::G; solvable::Int=0)::Tuple{SparseMatrixCSC, Vector{Symbol}, Vector{Symbol}} where G <: AbstractDFG
```

Comment:
I'm dropping Sparse for the default. Its silly to have a dense matrix. We can have getAdjacencyMatrixDense.